src/raftstore/store/store.rs

```diff
@@ -67,6 +68,7 @@ pub struct Store<T: Transport, C: PdClient + 'static> {
     store: metapb::Store,
     engine: Arc<DB>,
     sendch: SendCh<Msg>,
+    snapshot_rx: Receiver<Msg>,
```

Comment:
`tx` or `rx` is used for local temporary variable, use a more clear name is better.